### ISSUE TITLE
result after crf is 68.7

### ISSUE THREAD

**Issue by xd-429** [linyq2117/CLIP-ES#30]
Hello, the initial CAM generated by me is 70.78, and the result after crf is 68.7. Why is this?In addition, the time I use dCRF is much longer than the 0.2 hour mentioned in the paper. Why is this?
![image](https://github.com/user-attachments/assets/cd71f895-5890-4675-9107-a7bb9ec523f0)

**Comment by xd-429** [comment]
你好，请问论文的表1和表2都是用的 train.txt 得到吗？那table 3是如何得到的呢，训练集用的train.txt还是train_aug.txt呢？

**Comment by linyq2117** [comment]
Sorry for the late response.

The 70.78 mIoU is the result evaluated on `train.txt`, while 68.7 is for `train_aug.txt`. As mentioned in the table captions,, Table 1 and Table 2 are reported on `train.txt`, while Table 3 and Table 4 are reported on `train_aug.txt`. The time cost of dCRF is strange. I re-run the code and the time cost is normal in my case. Did you modify the code? If not, you may check your machine (e.g., is it busy?) or try to use the same workers (jobs=32) to ours.
<img width="928" alt="image" src="https://github.com/user-attachments/assets/1ceaeac5-8714-49a3-b76d-fc902b95c607">

**Comment by franziskaweber** [comment]
Hello,
as I had the same confusion I wanted to ask why you chose to train your model on the augmented training set but then reported the scores on the regular training set?

**Comment by linyq2117** [comment]
> Hello, as I had the same confusion I wanted to ask why you chose to train your model on the augmented training set but then reported the scores on the regular training set?

Thanks for your interest!

Our CLIP-ES is used to generate pseudo labels for segmentation, which is training-free. We evaluate the quality of pseudo masks in Table 1 on train set following previous works. Then, we use these pseudo masks to train semantic segmentation models(e.g., DeepLab), which is train on training set and evaluate on val set. Therefore, CLIP-ES can directly produce pseudo masks and doesn't need training.
